presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -80,6 +81,9 @@ class PrestoServer {
   }
 
  protected:
+  virtual void addAdditionalPeriodicTasks(
+      PeriodicTaskManager& periodicTaskManager);
```

Comment:
Can you make periodicTaskManager a member for Presto server as the other major presto server components? Then you don't need to pass periodic task manager here?